### PR TITLE
Improve us of service command

### PR DESCRIPTION
**Pull request by krhubert**

### CODE REVIEW

File: src/commands/service/compile.ts [hunk 1]
```diff
@@ -11,15 +11,15 @@ import deployer, {createTar} from '../../utils/deployer'
...
+  static description = 'Compile a service and upload its source to IPFS server'
 
   static flags = {
     ...Command.flags
   }
 
   static args = [{
-    name: 'SERVICE_PATH_OR_URL',
-    description: 'Path of the service or url to access it',
+    name: 'SERVICE',
```
Comment (NicolasMahe) [inline src/commands/service/compile.ts]: You need to rename `args.SERVICE_PATH_OR_URL` to `args. SERVICE` at line 31 (check for other lines)

File: src/commands/service/compile.ts [hunk 2]
```diff
@@ -11,15 +11,15 @@ import deployer, {createTar} from '../../utils/deployer'
...
+    description: 'Path or url ([https|mesg|ipfs]://) of a service',
```
Comment (NicolasMahe) [inline src/commands/service/compile.ts]: I don't think we are directly compatible with ipfs? Are we?

File: src/commands/service/delete.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export default class ServiceDelete extends Command {
 
   static flags = {
     ...Command.flags,
-    confirm: flags.boolean({description: 'Confirm delete', default: false})
+    confirm: flags.boolean({description: 'Automatic confirm deletion', default: false})
```
Comment (NicolasMahe) [inline src/commands/service/delete.ts]:
```suggestion
    confirm: flags.boolean({description: 'Confirm deletion', default: false})
```

File: src/commands/service/dev.ts [hunk 4]
```diff
@@ -17,8 +17,8 @@ export default class ServiceDev extends Command {
   }
 
   static args = [{
-    name: 'SERVICE_PATH',
-    description: 'Path of the service',
+    name: 'SERVICE',
+    description: 'Path or url ([https|mesg|ipfs]://) of a service',
```
Comment (NicolasMahe) [inline src/commands/service/dev.ts]:
```suggestion
    description: 'Path or url ([https|mesg]://) of a service',
```

File: src/commands/service/list.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@ import {Instance} from 'mesg-js/lib/api'
 import Command from '../../root-command'
 
 export default class ServiceList extends Command {
-  static description = 'List all instances'
+  static description = 'List instances'
```
Comment (NicolasMahe) [inline src/commands/service/list.ts]:
```suggestion
  static description = 'List running services'
```
Let's no speak about instance

File: src/commands/service/delete.ts [hunk 6]
```diff
@@ -21,7 +21,7 @@ export default class ServiceDelete extends Command {
   async run(): Promise<string[]> {
     const {argv, flags} = this.parse(ServiceDelete)
     if (!flags.confirm && !await cli.confirm('Are you sure?')) return []
-    this.spinner.start('Delete service')
+    this.spinner.start('Delete services')
```
Comment (NicolasMahe) [inline src/commands/service/delete.ts]:
```suggestion
    this.spinner.start('Deleting service(s)')
```

File: src/commands/service/doc.ts [hunk 7]
```diff
@@ -7,16 +7,16 @@ import {join} from 'path'
...
 
   static flags = {
     ...Command.flags,
     save: flags.boolean({char: 's', description: 'Save to default readme file'}),
   }
 
   static args = [{
-    name: 'SERVICE_PATH',
-    description: 'Path of the service',
+    name: 'SERVICE',
```
Comment (NicolasMahe) [inline src/commands/service/doc.ts]: You forget to change var name at line 25

File: src/commands/service/execute.ts [hunk 8]
```diff
@@ -42,7 +42,7 @@ export default class ServiceExecute extends Command {
 
     const task = service.tasks.find((x: any) => x.key === args.TASK)
     if (!task) {
-      throw new Error(`The task ${args.TASK} does not exists in the instance ${instanceHash}`)
+      throw new Error(`The task ${args.TASK} does not exist in mesg.yaml service`)
```
Comment (NicolasMahe) [inline src/commands/service/execute.ts]:
```suggestion
      throw new Error(`The task ${args.TASK} does not exist in service`)
```

File: src/commands/service/init.ts [hunk 9]
```diff
@@ -41,13 +41,13 @@ export default class ServiceInit extends Command {
     if (template) {
       return template
     }
-    this.spinner.start('Fetch the list of templates available')
+    this.spinner.start('Fetch the list of available templates')
```
Comment (NicolasMahe) [inline src/commands/service/init.ts]:
```suggestion
    this.spinner.start('Fetching templates')
```

File: src/commands/service/init.ts [hunk 10]
```diff
@@ -41,13 +41,13 @@ export default class ServiceInit extends Command {
...
     const templates = await this.fetchTemplates()
     this.spinner.stop()
     const {value} = (await prompt({
       type: 'list',
       name: 'value',
-      message: 'Choose the template you want to use',
+      message: 'Choose the template to use',
```
Comment (NicolasMahe) [inline src/commands/service/init.ts]:
```suggestion
      message: 'Select the template to use',
```

File: src/commands/service/stop.ts [hunk 11]
```diff
@@ -28,15 +28,14 @@ export default class ServiceStop extends Command {
...
       if (!await cli.confirm('Are you sure?')) return []
     }
-    this.spinner.start('Delete instance')
+    this.spinner.start('Stop running services')
     for (const hash of argv) {
-      this.spinner.status = hash
       await this.api.instance.delete({hash, deleteData: flags['delete-data']})
     }
-    this.spinner.stop(argv.join(', '))
+    this.spinner.stop()
```
Comment (NicolasMahe) [inline src/commands/service/stop.ts]:
```suggestion
    this.spinner.stop(argv.join(', '))
```
we need to keep important data in spinner for now

File: src/commands/service/stop.ts [hunk 12]
```diff
@@ -4,16 +4,16 @@ import cli from 'cli-ux'
...
 
 export default class ServiceStop extends Command {
-  static description = 'Stop a service by deleting a specific instance'
+  static description = 'Stop one or more running service'
 
   static flags = {
     ...Command.flags,
     'delete-data': flags.boolean({
-      description: 'Delete instances\' persistent data',
+      description: 'Delete runnign service persistent data',
```
Comment (antho1404) [inline src/commands/service/stop.ts]:
```suggestion
      description: 'Delete running service persistent data',
```

**Pull request merged**